.travis.yml

```diff
@@ -1,8 +1,18 @@
 language: python
+
+python:
+  - "2.6"
+  - "2.7"
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+  - "nightly"
```

Comment:
We probably want to [allow failures](https://docs.travis-ci.com/user/customizing-the-build/#Rows-that-are-Allowed-to-Fail) on the nightly version, since it's not stable.